build/build-pr-ci.yml

```diff
@@ -0,0 +1,76 @@
+variables:
+  buildConfiguration: Release
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    exclude:
+      - '**.md'
+
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    exclude:
+      - '**.md'
+
+continueOnError: false
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core SDK 6 LTS'
+    inputs:
+      version: '6.x'
+
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core SDK 7 STS'
+    inputs:
+      version: '7.x'
```

Comment:
This should be 8.x. Generally speaking, we should avoid depending on STS releases when possible.

Reply:
Yes, it should have been 8, updated.